.env-template

```diff
@@ -7,8 +7,8 @@ ELASTICSEARCH_PROTOCOL=https
 ELASTICSEARCH_INDEX=<set to override - defaults to latest complete collection index>
 
 # NetX configuration
-NETX_BASE_URL=<url_for_netx_instance>
 NETX_API_TOKEN=<api_token_from_netx>
+REACT_APP_NETX_BASE_URL=<url_for_netx_instance>
```

Comment:
Switching the NetX Base URL to a React App environment variable since configurations between UAT and Production NetX are not the same.